pkg/pillar/cmd/zedrouter/pbr.go

```diff
@@ -485,3 +489,47 @@ func AddOverlayRuleAndRoute(bridgeName string, iifIndex int,
 	}
 	return nil
 }
+
+// AddFwMarkRuleToDummy : Create a ip rule that sends packets marked with given mark
+// out of interface with given index.
+func AddFwMarkRuleToDummy(fwmark uint32, iifIndex int) error {
+
+	r := netlink.NewRule()
+	myTable := FreeTable + iifIndex
+	r.Table = myTable
+	r.Mark = int(fwmark)
+	// XXX Explain this magic number
```

Comment:
Which magic number? What is the explanation?

Reply:
I should have removed this comment. Will remove.